js-intro.js

```diff
@@ -50,7 +61,8 @@ var myMessage = "Learning is fun!"
 
 
 // 3b. Write the code that logs each letter of the message using map.
-
+var mapMessage = myMessage.map((value => value));
+console.log(mapMessage);
 
 
```

Comment:
Just need to add a split to your code to take the variable from a string into an array
var mapMessage = myMessage.split("").map(value => value);